datadog_checks_base/pyproject.toml

```diff
@@ -0,0 +1,104 @@
+[build-system]
+requires = [
+    "hatchling>=0.11.2",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-checks-base"
+description = "The Datadog Check Toolkit"
+readme = "README.md"
+license = "BSD-3-Clause"
+keywords = [
+    "datadog",
+    "agent",
+    "checks",
+]
+authors = [
+    { name = "Datadog", email = "packages@datadoghq.com" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.8",
+    "Topic :: System :: Monitoring",
+]
+dynamic = [
+    "version",
+]
+
+[project.optional-dependencies]
+db = [
+    "mmh3==2.5.1; python_version < '3.0'",
+    "mmh3==3.0.0; python_version > '3.0'",
+]
+deps = [
+    "binary==1.0.0",
+    "cachetools==3.1.1; python_version < '3.0'",
+    "cachetools==4.2.4; python_version > '3.0'",
+    "contextlib2==0.6.0; python_version < '3.0'",
+    "cryptography==3.3.2; python_version < '3.0'",
+    "cryptography==3.4.8; python_version > '3.0'",
+    "ddtrace==0.32.2; sys_platform == 'win32' and python_version < '3.0'",
+    "ddtrace==0.53.2; sys_platform != 'win32' or python_version > '3.0'",
+    "enum34==1.1.10; python_version < '3.0'",
+    "immutables==0.16; python_version > '3.0'",
+    "ipaddress==1.0.23; python_version < '3.0'",
+    "prometheus-client==0.9.0",
+    "protobuf==3.7.0",
+    "pydantic==1.8.2; python_version > '3.0'",
+    "python-dateutil==2.8.2",
+    "pywin32==228; sys_platform == 'win32' and python_version < '3.0'",
+    "pywin32==300; sys_platform == 'win32' and python_version > '3.0'",
+    "pyyaml==5.4.1",
+    "requests-toolbelt==0.9.1",
```

Comment:
This package is `requests_toolbelt` in the [requirements.in](https://github.com/DataDog/integrations-core/pull/11301/files#diff-dfb35ae8434d1c8178f18ebc7c8a58dbb1c75d790b7eaa3a3b0566d085e63512L32) file. Does the new pyproject.toml take the [pypi](https://pypi.org/project/requests-toolbelt/) package name rather than the imported package name?

Reply:
Either is fine but yes I put the PyPI name 👍